### DISCUSSION TITLE
Microphone is not working

### DISCUSSION THREAD

**Discussion by matheuswm82** [Genymobile/feedback#91] [Genymotion Desktop] [answered]
How can I test my microphone on mac m2? i'm emulating android 13.0 api 33 (google pixel C). the microphone is not working. I tried to find to find all acess, and all allowed.

**Comment by jmaneyrol69** [comment, marked as answer]
@matheuswm82 we are terribly sorry but this discussion board is for feature requests **only** - it is not suitable for technical assistance requests.
Please use our help center at https://support.genymotion.com/hc/en-us/ for this purpose.
Thank you.
